stack_definition.json

```diff
@@ -261,8 +262,58 @@
       ]
     },
     {
-      "name": "4 - WatsonX SaaS services",
-      "version_locator": "1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc.c237751c-d3dd-4c23-b0b1-ba726d30c8e9-global",
+      "name": "4 - Databases for Elasticsearch",
+			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.c5ea982b-011a-46c8-a49d-0de273f40b46-global",
+			"inputs": [
+				{
+					"name": "use_existing_resource_group",
+					"value": true
+				},
+				{
+					"name": "resource_group_name",
+					"value": "ref:../../members/1 - Account Infrastructure Base/outputs/workload_resource_group_name"
+				},
+				{
+					"name": "region",
+					"value": "ref:../../inputs/region"
+				},
+				{
+					"name": "prefix",
+					"value": "ref:../../inputs/prefix"
+				},
+				{
+					"name": "plan",
+					"value": "platinum"
```

Comment:
I think that should be a parameter from the top o the stack (and even there the lower plan as default may be more appropriate). Setting to platinum by default would lead to unpleasant surprises in billing.